src/WebApiContrib.Core.Formatter.Csv/CsvFormatterOptions.cs

```diff
@@ -7,5 +7,7 @@ public class CsvFormatterOptions
         public string CsvDelimiter { get; set; } = ";";
 
         public string Encoding { get; set; } = "ISO-8859-1";
+        
+        public bool UseJsonPropertyJsonIgnoreAttributes { get; set; } = true;

```

Comment:
I still hate this, I think this goes against any configuration model that exists in either these projects here and the MVC framework itself

if you want to make it configurable, my suggestion is to keep this hidden and just enable by default. Then, instead of adding a rather bizarre `UseJsonPropertyJsonIgnoreAttributes` bool property, we should allow building strategies for object serialization, that will be responsible for discovering properties and reading the values.

This is - IMHO - needed anyway, for scenarios like localization. If you want to disable JSON.NET attributes you'd just plug in a different strategy then.

I think just adding a bool to cover this case is a design shortcut.

Reply:
^Totally open to see other implementations, I have no strong feeling about it and won't complain at all as long as there is a possibility (a bool or other mechanism as you mentioned) to turn on/off the JSON attributes support.

Reply:
see your point :) I'll remove the configuration and have it turned on per default.

Reply:
OK great, so let's merge this and come up with a more flexible design then. Like I mentioned this is something that I already had on my mind with regards to i.e. localization.

> as long as there is a possibility (a bool or other mechanism as you mentioned) to turn on/off the JSON attributes support.

and yet you use the MVC framework where you can't turn it off 😝😄

Reply:
Yes, I do use MVC, but that doesn't mean I wouldn't prefer having more options to turn on/off some features. And it has to be said that there is a ton of extensibility points in MVC so I can customize pretty much anything I want.